include/ruby/internal/fl_type.h

```diff
@@ -111,13 +111,6 @@
 #define RB_OBJ_FREEZE_RAW    RB_OBJ_FREEZE_RAW
 #define RB_OBJ_FROZEN        RB_OBJ_FROZEN
 #define RB_OBJ_FROZEN_RAW    RB_OBJ_FROZEN_RAW
-#define RB_OBJ_INFECT        RB_OBJ_INFECT
-#define RB_OBJ_INFECT_RAW    RB_OBJ_INFECT_RAW
-#define RB_OBJ_TAINT         RB_OBJ_TAINT
-#define RB_OBJ_TAINTABLE     RB_OBJ_TAINTABLE
-#define RB_OBJ_TAINTED       RB_OBJ_TAINTED
-#define RB_OBJ_TAINTED_RAW   RB_OBJ_TAINTED_RAW
-#define RB_OBJ_TAINT_RAW     RB_OBJ_TAINT_RAW
 #define RB_OBJ_UNTRUST       RB_OBJ_TAINT
```

Comment:
@nobu should we also remove `RB_OBJ_UNTRUST` and `RB_OBJ_UNTRUSTED`?

Reply:
These 2 replace different names.